.flake8

```diff
@@ -5,3 +5,4 @@ max-line-length = 88
 extend-ignore =
     # See https://github.com/PyCQA/pycodestyle/issues/373
     E203,
+ignore = W605
```

Comment:
why is this needed?

Reply:
"Return a mapping `{y: {x s.t. y \in f(x)}}, where `x \in it`. `"

when pre-commiting:
`gamla/graph.py:96:40: W605 invalid escape sequence '\i'`